### PR TITLE
`<Notification/>`- adding status icons 

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Adding icons to `Notification`. 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

File: src/Notification/test/Notification.visual.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { THEMES } from '../constants';
```
Comment (EgoziE) [inline src/Notification/test/Notification.visual.js:1]: Maybe you can try using `storyOfAllPermutations`

File: src/Notification/Notification.scss [hunk 2]
```diff
@@ -5,6 +5,11 @@
   position: relative;
 }
 
+.iconStyling{
```
Comment (EgoziE) [inline src/Notification/Notification.scss:8]: Is it much work to migrate this component to Stylable?

Comment (TalZa1810) [reply src/Notification/Notification.scss:8]: I'll do it in a different PR. I want to merge the Visuals to have this as a baseline for the changes :)

**Comment by TalZa1810** [comment]
Hi @milkyfruit ,
I've added here the icons for the `success`, `error` and `warning` themes. 
Let me know if I can merge this :)

**Pull request merged**